lib/philomena_web/templates/layout/app.html.slime

```diff
@@ -19,7 +19,7 @@ html lang="en"
     meta name="theme-color" content="#618fc3"
     meta name="format-detection" content="telephone=no"
     = csrf_meta_tag()
-    script type="text/javascript" src=Routes.static_path(@conn, "/js/app.js") async="async"
+    script type="module" src=Routes.static_path(@conn, "/assets/js/app.js")
```

Comment:
Is module needed? Why no longer async? Why did the path change?

Reply:
> Is module needed? Why no longer async? Why did the path change?

vite seems to operate on modules for its hot reload functionality. No longer async might have just been a leftover derp from troubleshooting it. New path because that's where vite writes the assets and it seemed to be the path of least resistance to just roll with it.